azure-pipelines.yml

```diff
@@ -111,3 +111,50 @@ jobs:
           additionalDockerComposeFiles: docker-compose.eval.yml
           projectName: $(Build.Repository.Name)
           dockerComposeCommand: up --remove-orphans --abort-on-container-exit --exit-code-from eval
+
+  - job: publish
```

Comment:
In this case it makes sense either way, but in general - AzurePipelines has separate tool for Deployment, in case we would want to have more control over it